src-electron/main-process/modules/backend.js

```diff
@@ -295,7 +295,7 @@ export class Backend {
           const baseUrl =
             net_type === "testnet"
               ? "https://lokitestnet.com"
```

Comment:
might as well update this one too
```suggestion
              ? "https://testnet.oxen.observer/"
```

Reply:
Done 👍

Reply:
Cheers for the contribution!